README.md

```diff
@@ -1,2 +1,21 @@
 # hw01
 Repository for homework 01
+
+#**Sarah Jensen**
+##Intro
+###My name is Sarah, and I am a 2018 graduate of St. Norbert College where I earned my B.A. in Psychology and French. I study moral psychology and am interested in how children develop their sense of right and wrong based on their social environments. 
+
+##Fun Facts
+* I play the flute
+* I tap danced for 16 years
+* I know how to juggle
+* I like to eat pizza
+
+![pizza](https://www.soccerunityproject.com/uploads/3/9/0/4/39046753/s284287166891139567_p50_i1_w1920.jpeg)
+
+###Bio and Photo
+*Here is a brief biography from the lab I worked in during undergrad.*
+
+[linked bio and pic] (https://ddmlab.weebly.com/sarah-jensen.html)
```

Comment:
You really don't need to indicate a hyperlink with the term "link". You could have even made the "Bio and Photo" in the header a hyperlink.
Also, you really need to pay attention to what does and does not need a space (as we all do, it appears).